invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v6.0.0.json

```
@@ -1046,14 +1049,14 @@
       "metadata.description",
       "metadata.formats",
       "metadata.funding.award.identifiers.identifier",
-      "metadata.funding.award.funder.name",
+      "metadata.funding.award.acronym.text",
+      "metadata.funding.funder.name",
```

```suggestion
      "metadata.funding.award.number",
      "metadata.funding.funder.name",
```

minor: let's also add this one in the mix